src/main/java/com/sharp/vn/its/management/repositories/TaskRepository.java

```diff
@@ -6,23 +6,30 @@
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.domain.Specification;
 import org.springframework.data.jpa.repository.Query;
-import org.springframework.data.repository.query.Param;
+
+import java.util.List;
 
 /**
  * The interface Task repository.
  */
 @Transactional
-public interface TaskRepository extends BaseJpaRepository<TaskEntity, Long> {
+public interface TaskRepository extends BaseJpaRepository<TaskEntity, Long>, TaskRepositoryCustom {
 
     /**
      * Find all page.
      *
-     * @param spec the spec
+     * @param spec     the spec
      * @param pageable the pageable
      * @return the page
      */
     Page<TaskEntity> findAll(Specification<TaskEntity> spec, Pageable pageable);
 
+    /**
+     * Exists by system id boolean.
+     *
+     * @param systemId the system id
+     * @return the boolean
+     */
```

Comment:
Why has the format of this file changed

Reply:
I import java.util.List for list year and used shortcut Ctr+Alt+O to remove unnecessary library import so the format of this file changed